packages/web/src/i18n.ts

```diff
@@ -0,0 +1,51 @@
+import i18n from "i18next";
+import { initReactI18next } from "react-i18next";
+
+i18n
+  .use(initReactI18next) // passes i18n down to react-i18next
+  .init({
+    resources: {
+      en: {
```

Comment:
I'm good with keeping these here for now 👍 We'll look into moving these into the common package so we can circumvent the weird CI issue.